client/src/components/RuleBuilder/SavedRulesSelector.test.js

```diff
@@ -1,15 +1,19 @@
 import Vue from "vue";
 import { mount } from "@vue/test-utils";
 import SavedRulesSelector from "components/RuleBuilder/SavedRulesSelector";
+import { getNewAttachNode } from "../../../tests/jest/helpers";
```

Comment:
import { getNewAttachNode } from "jest/helpers";
```suggestion
import { getNewAttachNode } from "jest/helpers";
```